client/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,203 @@
+import React, { Component } from "react";
+import "./Layout.css";
+import { Route, Redirect } from "react-router-dom";
+import Navbar from "../../components/Navbar/Navbar"
+import Footer from "../../components/Footer/Footer"
+import SplashPage from "../../pages/SplashPage/SplashPage";
+import BookATripPage from "../../pages/BookATripPage/BookATripPage";
+import CommunityPage from "../../pages/CommunityPage/CommunityPage";
+import CommunityPageGroup from "../../pages/CommunityPage/CommunityPageGroup";
+import ForumPageHashtag from "../../pages/ForumPage/ForumPageHashtag";
+import ForumPageTopic from "../../pages/ForumPage/ForumPageHashtagTopic";
+import ForumPageDiscover from "../../pages/ForumPage/ForumPageDiscover";
+
+import CommunityPagePeople from "../../pages/CommunityPage/CommunityPagePeople";
+import ForumPage from "../../pages/ForumPage/ForumPage";
+import DirectoryPeople from "../../pages/DirectoryPage/DirectoryPeople";
+import DirectoryGroup from "../../pages/DirectoryPage/DirectoryGroups";
+import HomePage from "../../pages/HomePage/HomePage";
+import MemberProfilePage from "../../pages/MemberProfilePage/MemberProfilePage";
+import TripDetail from "../../pages/TripDetail/TripDetail";
+import OnboardingPage from "../../pages/OnboardingPage/OnboardingPage";
+import LoginPage from "../../pages/LoginPage/LoginPage";
+
+// Class Based React Component
+export default function Layout(props) {
+    console.log(props)
+    return (
+        <div className="Layout">
+            {props.logged_in ? (
+                <Route
+                    path="/"
+                    exact
+                    render={routerProps => (
+                        <main>
+                            <Navbar to={`/member-page/${props.userid}`} profileImage={props.image} page={5}></Navbar>
```

Comment:
We'll need some way to pass in the page number from App

Reply:
Page number determined by useLocation from react-router-dom.